buildlib/tools/builds.sh

```diff
@@ -448,6 +448,23 @@ then
 			'build_armclang')
 fi
 
+# Check if a specific test was requested as an argument
+if [ "$1" ]; then
+	specific_test=$1
+
+	# Verify the requested test exists in the list
+	if [[ " ${tests[@]} " =~ " ${specific_test} " ]]; then
+		tests=("$specific_test")
+		num_tests=1
```

Comment:
maybe just run the test here and exit, no need to go to the loop that updates Azure progress etc

Reply:
changed it